easybuild/easyconfigs/r/Ragout/Ragout-2.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'Ragout'
+version = '2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://fenderglass.github.io/%(name)s/'
+description = """Ragout (Reference-Assisted Genome Ordering UTility) is a tool for chromosome assembly using multiple
+ references. Given a set of assembly fragments (contigs/scaffolds) and one or multiple related references (complete or
+ draft), it produces a chromosome-scale assembly (as a set of scaffolds)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/fenderglass/%(name)s/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['85d57ab050572332eb5879e4ac98acf50c0de373b1a1a01876e60b6e222f1486']
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('Sibelia', '3.0.6'),
+]
+
+runtest = 'test'
+
+files_to_copy = ['docs', 'examples', 'lib', '%(namelower)s', 'scripts', '%(namelower)s.py', 'LICENSE', 'README.md']
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['%(namelower)s.py'],
```

Comment:
@SethosII I would also check for the two binaries that are being compiled, i.e. `lib/ragout-maf2synteny` and `lib/ragout-overlap`

Reply:
@boegel I changed this.